.goreleaser.yml

```diff
@@ -1,18 +1,22 @@
 # This is an example .goreleaser.yml file with some sane defaults.
 # Make sure to check the documentation at http://goreleaser.com
+
+env:
+  - CGO_ENABLED=0
+
 before:
   hooks:
     # You may remove this if you don't use go modules.
     - go mod download
     # you may remove this if you don't need go generate
     - go generate ./...
 builds:
-  - env:
-      - CGO_ENABLED=0
-    goos:
+  - goos:
```

Comment:
why did you replaced `goos:`  by `- goos:` ?
We don't support Windows and Apple, do we ?

Reply:
We can release the binary of these arcs. Please see also https://github.com/fybrik/json-schema-generator/.goreleaser

Reply:
It's a vault plugin, so to me it makes sense to build to all platforms. Although I'm not sure about the status of the implementation of the plugin, we talked about out of cluster vault and multiple remote clusters but I don't remember whether you implemented those or kept it aside because it wasn't prioritized.

Reply:
@roee88 I did not experiment with this case. IIUC from  https://learn.hashicorp.com/tutorials/vault/plugin-backends#setup-vault it seems using a plugin in an already running server involves reloading the configuration into vault server. but I did not try it...